frontend/scss/globals/_g-header.scss

```diff
@@ -14,7 +14,16 @@
   margin-left: -50vw;
   left: 50%;
   right: 50%;
+}
 
+.g-header:has([aria-expanded="true"]) ~ main::after {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  z-index: map-get($zindexs, 'stickyFilters');
   @supports (backdrop-filter: blur(5px)) or (-webkit-backdrop-filter: blur(5px)) {
     -webkit-backdrop-filter: blur(5px);
     backdrop-filter: blur(5px);
```

Comment:
I thought that I would have blur the background using JS, but I was able to do it in CSS!